.github/workflows/ci.yaml

```diff
@@ -166,7 +166,7 @@ jobs:
       - name: Deploy build to rustup-builds bucket for release team
         if: github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.mode == 'release'
         run: |
-          aws --debug s3 cp --recursive dist s3://rustup-builds/${{ github.sha }}
+          aws --debug s3 cp --recursive dist s3://rustup-builds/${{ github.sha }}/dist
```

Comment:
This file is generated (see notice near the top)... should probably change the source?

Reply:
I did. 😉 I added to the change to `windows-builds-template.yaml` and then ran the script to apply it to `ci.yaml`. Should be all good.

Reply:
Ahh, sorry for the noise!